plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/IFordiacModifiablePreviewChange.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Primetals Technologies Austria GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Mario Kastner
+ *     - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+
+package org.eclipse.fordiac.ide.typemanagement.refactoring;
+
+import org.eclipse.fordiac.ide.model.typelibrary.TypeEntry;
+
+public interface IFordiacModifiablePreviewChange {
+
+	void updatePackageName(String newName);
```

Comment:
Why do you need the interface ? 
It is not very generic.
i would rather name the method something like "update text" or "getText".
And the interface i would name something like TextEditPreview (maybe you find a better name).
But nevertheless i like the idea of make it reusable